### ISSUE TITLE
Issue with moving columns in Firefox browser

### ISSUE THREAD

**Issue by dskumar001** [neokoenig/jQuery-gridmanager#64]
Hi,

When I move column for the 2nd time the column is showing all the way at top.
For example when i move column which is at the bottom, it looks like we are missing that column when we try to move.

This is happening only in firefox browser and works fine in Chrome and IE
Please find the screenshot below

![movecolumnissue](https://cloud.githubusercontent.com/assets/4333359/8125174/1714809c-1101-11e5-9435-2803e682a041.png)

Thanks

**Comment by neokoenig** [comment]
Yeah definitely a firefox specific bug. Not come across this before.
